ubi8-openjdk-11-runtime.yaml

```diff
@@ -0,0 +1,50 @@
+# This is an Image descriptor for Cekit
+
+schema_version: 1
+
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
+name: &name "ubi8/openjdk-11"
+version: &version "1.3"
+description: "Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 11"
```

Comment:
We will need to re-word this to distinguish it from the existing images. I suggest initially adding at the end " runtime", and removing "Source To Image (S2I) " at the front